src/subcommand/server.rs

```diff
@@ -1127,6 +1136,8 @@ impl Server {
     let next = index.get_inscription_id_by_sequence_number(entry.sequence_number + 1)?;
 
     let children = index.get_children_by_inscription_id(inscription_id)?;
+    let end_index = usize::min(children.len(), 20);
```

Comment:
Which is the best place to store the per-page count as a constant?

Reply:
I think just putting it here is fine, we do the same of the normal inscriptions pagination.